### DISCUSSION TITLE
Progress DB - Driver

### DISCUSSION THREAD

**Discussion by Hogeback** [mtxr/vscode-sqltools#1288] [New Drivers]
Hello,

I'll ask you a cheeky question.

Is there a way to create a driver for a Progress DB?

I don't think I can manage to develop a driver myself.

I would be pleased about a positive feedback.

Best regards
Tim

**Comment by gjsjohnmurray** [comment]
You may be able to bring such a driver into existence by offering to sponsor its development by someone who has the relevant skills. If you're interested in doing this I suggest you post an initial message here.

**Comment by Yahud100** [comment]
I’ve installed the CockroachDB driver (cockroachdb@1.0.3) and confirmed it with yarn list, but SQLTools in the VS Code extension development host still shows "Driver PostgreSQL is not installed."

Steps Taken:
Added cockroachdb to package.json.
Installed using yarn.
Verified installation.

Questions:

Is there a way to include a PostgreSQL driver (like CockroachDB) by default in SQLTools?
What steps are needed to ensure the CockroachDB driver is recognized?

have  installed cockroach version 1.0.3 and my yarn version  is 1.22.22
